src/modules/lesson/controllers/lesson-hashtag.controller.ts

```diff
@@ -0,0 +1,195 @@
+import {
+  Body,
+  Controller,
+  Delete,
+  Get,
+  HttpStatus,
+  Param,
+  Post,
+  Put,
+} from '@nestjs/common';
+import {
+  ApiOkResponse,
+  ApiOperation,
+  ApiTags,
+  PickType,
+} from '@nestjs/swagger';
+import { ModelName } from '@src/constants/enum';
+import { ApiFailureResponse } from '@src/decorators/api-failure-response.decorator';
+import { ApiSuccessResponse } from '@src/decorators/api-success-response.decorator';
+import { BearerAuth } from '@src/decorators/bearer-auth.decorator';
+import { SetModelNameToParam } from '@src/decorators/set-model-name-to-param.decorator';
+import { UserLogin } from '@src/decorators/user-login.decorator';
+import { IdRequestParamDto } from '@src/dtos/id-request-param.dto';
+import { JwtAuthGuard } from '@src/guards/jwt-auth.guard';
+import { OptionalJwtAuthGuard } from '@src/guards/optional-auth-guard';
+import { PrismaHelper } from '@src/modules/core/database/prisma/prisma.helper';
+import { CreateManyHashtagDto } from '@src/modules/hashtag/dtos/create-many-hashtag.dto';
+import { LessonHashtagParamDto } from '@src/modules/hashtag/dtos/hashtag-param.dto';
+import { UpdateHashtagDto } from '@src/modules/hashtag/dtos/update-hashtag.dto';
+import { UpdateManyHashtagDto } from '@src/modules/hashtag/dtos/update-many-hashtag.dto';
+import { LessonHashtagEntity } from '../entities/lesson-hashtag.entity';
+import { LessonEntity } from '../entities/lesson.entity';
+import { LessonHashtagService } from '../services/lesson-hashtag.service';
+
+@ApiTags('과제의 해시태그')
+@Controller(':id/hashtags')
+export class LessonHashtagController {
+  constructor(
+    private readonly lessonHashtagService: LessonHashtagService,
+    private readonly prismaHelper: PrismaHelper,
+  ) {}
+
+  @ApiOperation({ summary: '과제 해시태그 생성' })
+  @ApiOkResponse({ type: PickType(LessonEntity, ['hashtags']) })
+  @ApiFailureResponse(HttpStatus.FORBIDDEN, 'You do not have access to ~')
+  @ApiFailureResponse(HttpStatus.NOT_FOUND, "~ doesn't exist id in ~")
+  @BearerAuth(JwtAuthGuard)
+  @Post()
+  async createHashtag(
+    @Param()
+    @SetModelNameToParam(ModelName.Lesson)
+    param: IdRequestParamDto,
+    @Body() { hashtags }: CreateManyHashtagDto,
+    @UserLogin('id') memberId: number,
+  ) {
+    await this.prismaHelper.validateOwnerOrFail(ModelName.Lesson, {
+      id: param.id,
+      memberId,
+    });
+
+    const createdHashtags = await this.lessonHashtagService.createHashtag(
+      hashtags,
+      param.id,
+    );
+    return { hashtags: createdHashtags };
```

Comment:
```suggestion
    );
    
    return { hashtags: createdHashtags };
```
진짜 사소한데 습관을 들이면 좋습니다!

Reply:
꼼꼼하게 하도록 하겠습니다!